windows_mobile/ShowProofDlg.qml

```diff
@@ -77,6 +77,11 @@ Item {
         isTxDetailView = transactionDetail.visible
         transactionDetail.visible = false
         showProofDlg.visible = true
+        if (isExport) {
```

Comment:
Export message should be more clear about that file is written.  Title text is good, but please add at the text something like this:
"Transaction proof is exported to   /storage/emulated/0/Android/data/mw.mwc.wallet/files/Download/XXXXXXXXXX.proof"

Reply:
@bayk There is already a location field in proof detail dialog which shows the path of the proof file so I guess, we don't need to show the duplication thing in the proof detail.